app/client/ui/PagePanels.ts

```diff
@@ -79,12 +90,15 @@ export function pagePanels(page: PageContents) {
     leftOverlap.set(false);
   });
 
-  // When url changes, we must have navigated; close the left panel since if it were open, it was
-  // the likely cause of the navigation (e.g. switch to another page or workspace).
+  // When url changes, we must have navigated;
+  //   - close the left panel since if it were open, it was the likely cause of the navigation
+  //     (e.g. switch to another page or workspace).
+  //   - reset the focus switcher to behave like a normal browser navigation (lose focus).
   const sub2 = subscribe(isNarrowScreenObs(), urlState().state, (use, narrow, state) => {
     if (narrow) {
       left.panelOpen.set(false);
     }
+    regionFocusSwitcher?.reset();
```

Comment:
It doesn't seem like you need the question-mark anywhere in this function, since `regionFocusSwitcher` is created unconditionally.